guide/src/conversions/tables.md

```diff
@@ -37,6 +37,7 @@ The table below contains the Python type and the corresponding function argument
 | `decimal.Decimal` | `rust_decimal::Decimal`[^5] | -                    |
 | `ipaddress.IPv4Address` | `std::net::IpAddr`, `std::net::IpV4Addr` | - |
 | `ipaddress.IPv6Address` | `std::net::IpAddr`, `std::net::IpV6Addr` | - |
+| `os.PathLike ` | `PathBuf`, `Path`              | `&PyString`, `&PyUnicode` |
 | `pathlib.Path` | `PathBuf`, `Path`              | `&PyString`, `&PyUnicode` |
```

Comment:
Isn't `pathlib.Path` also an `os.PathLike`, i.e. doesn't the first line basically include the second one?

Reply:
Indeed. I left the two lines for explictness because I assumed a lot of people are not used to `os.PathLike`. If you prefer I can remove the `pathlib.Path` line.

Reply:
Personally, I would merge them and maybe use "`pathlib.Path`, `os.PathLike`" for the first column to stay explicit. But let's see what the other maintainers think.

Reply:
I was thinking of upending this table anyway and describing it from the Rust types (what they accept and what they convert to), so I'm ok with leaving this as two rows for now.